pyglmnet/pyglmnet.py

```diff
@@ -780,6 +785,10 @@ def fit(self, X, y):
             if callable(self.callback):
                 self.callback(beta)
 
+        if self.n_iter_ == self.max_iter:
+            logger.warning(
+                "Reached max number of iterations without conversion.")
```

Comment:
conversion -> convergence?

Can you also add a test that checks this warning is raised?